gsl_projet/static/css/projet_list_filters.css

```
@@ -81,6 +88,34 @@
 }
 
 .filter-dropdown-button-active {
-    border: 2px solid var(--artwork-major-blue-cumulus-active);
-    padding-top: 6px;
+  border: 2px solid var(--artwork-major-blue-cumulus-active);
+  padding-top: 6px;
+}
+
+#filter-status-button > button {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+#filter-status-button .filter-content {
+  min-width: auto;
+  gap: 6px;
+}
+
+#filter-status-button label[for="id_status_0"] {
+  color: var(--text-default-success);
+}
+
+#filter-status-button label[for="id_status_1"],
```

J'aurais utilisé les valeurs des statuts plutôt que leur position dans la liste : comme ça on n'a pas besoin de modifier le CSS en comptant sur nos doigts si on veut changer l'ordre d'affichage et/ou ajouter des statuts. 

Donc par ex. ici 

```suggestion
#filter-status-button label[for="id_status_en_construction"],
```

avec une modif plus bas pour changer la logique de génération des ID des champs.